lib/api/controllers/routerController.js

```diff
@@ -276,7 +275,7 @@ function executeFromRest(params, request, response) {
     delete request.params.index;
   }
 
-  _.forEach(request.params, function (value, param) {
+  _.forEach(request.params, function bindParams (value, param) {
```

Comment:
Is it necessary to name this one ? Shouldn't we convert it to an arrow function instead?

Moreover, it may be the right time to get rid of this unnecessary call to lodash, replacing it with something like this:

```js
Object.assign(request.body, request.params);
```